persistence/jdbc/src/test/java/org/infinispan/persistence/jdbc/stringbased/StringStoreWithManagedConnectionFunctionalTest.java

```diff
@@ -13,30 +10,10 @@
 @Test(groups = "functional", testName = "persistence.jdbc.stringbased.StringStoreWithManagedConnectionFunctionalTest")
 public class StringStoreWithManagedConnectionFunctionalTest extends AbstractStringBasedCacheStore {
 
-    private ManagedConnectionFactoryConfiguration customFactoryConfiguration;
-
-    public StringStoreWithManagedConnectionFunctionalTest() {}
-
-    @SuppressWarnings("unused")
-    public StringStoreWithManagedConnectionFunctionalTest(ManagedConnectionFactoryConfiguration customFactoryConfiguration) {
-        this.customFactoryConfiguration = customFactoryConfiguration;
```

Comment:
This was previously required by the `jdg-functional-tests` I think. Are all of those tests now in the Infinispan repo? and is this definitely not needed?

Reply:
@ryanemerson you are right, but now I change the approach that I'm using on `jdg-functional-tests`, I'm overriding the methods that I need during the instantiation. 
On `jdg-functional-tests` now i'm using:
```
new StringStoreWithManagedConnectionFunctionalTest() {

            @Override
            protected ConnectionFactory getConnectionFactory(JdbcStringBasedStoreConfigurationBuilder storeBuilder) {
                ...
            }

            @Override
            public void setTableManipulation(JdbcStringBasedStoreConfigurationBuilder storeBuilder) {
               ...
            }
```